server/src/concepts/user/model/UserValidation.js

```diff
@@ -0,0 +1,30 @@
+const Joi = require("joi-oid");
+Joi.objectId = require("joi-objectid")(Joi);
+
+const validateUser = (user) => {
+  const schema = Joi.object({
+    id: Joi.objectId().required(),
+
+    email: Joi.string()
+      .min(6)
+      .max(255)
+      .required()
+      .email({ minDomainSegments: 2, tlds: { allow: ["com", "net"] } }),
```

Comment:
If I read docs correctly, for now you allow only .com and .net domains, I do not think that is a good idea, there are many more domains. Maybe just remove tlds option or just do allow: true to include all email from IANA registry. https://joi.dev/api/?v=17.6.0#stringemailoptions